app/connectors/securestorage/SecureStorage.scala

```diff
@@ -151,5 +151,6 @@ trait SecureStorage {
   def objectStorage[T](clazz : Class[T])(implicit f : Format[T], ec : ExecutionContext) =
     new SecureStorageObjectInterface[T](this, clazz)
 
+  //scalastyle:off method.name
```

Comment:
I think you need to switch this back on after you've switched it off. Or use scalastyle:ignore at the end of the line applicable.

Reply:
I've switched it back on after the offending method